src/api/query/accountInfo.ts

```diff
@@ -0,0 +1,38 @@
+import { state } from '@/store/state';
+import { getters } from '@/store/getters';
+import { AccountInfo, Address, Balance } from '@/api/models';
+import { Ref } from 'vue';
+
+export async function subscribeAccountInfo(account: Address, ref: Ref<AccountInfo>) {
```

Comment:
right now the account info and balance subscription function are all separated. Do you think there is a way to simplify this by using the Vue Reactivity API and make it look like this https://github.com/hoonsubin/apps/blob/master/packages/react-hooks/src/useAccounts.ts?

Reply:
Let me see!